Sources/SwiftDocC/Utility/FeatureFlags.swift

```diff
@@ -14,12 +14,6 @@ public struct FeatureFlags: Codable {
     /// (usually experimental) behavior in Swift-DocC.
     public static var current = FeatureFlags()
     
-    /// Whether or not experimental language support for Objective-C is enabled.
-    ///
-    /// This can be enabled on the command-line by passing `--enable-experimental-objective-c-support`
-    /// to docc.
-    public var isExperimentalObjectiveCSupportEnabled = false
```

Comment:
Should we leave this but deprecate it?

Reply:
I guess it's possible for clients of the `SwiftDocC` library to set this property directly, so keeping it temporarily makes sense. I added a deprecation notice.